infrastructure/main.tf

```
@@ -10,7 +10,8 @@ locals {
 
 module "reform-blob-router-db" {
   source             = "git@github.com:hmcts/cnp-module-postgres?ref=master"
-  product            = var.component
+  product            = var.product
+  component          = var.component
```

the previous name was only using the component so we need to keep using it by providing the name override
```suggestion
  component          = var.component
  name               = var.component
```